jax/scipy/sparse/test_gmres.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
```

Comment:
This test file should go in `tests/`, for consistency with JAX's other tests.

Reply:
Ok, I'll write parameterized tests there.